src/controllers/stallController.ts

```diff
@@ -95,6 +95,14 @@ async function showStall(req: Request, res: Response, next: NextFunction) {
   }
 }
 
+async function showStalls(req: Request, res: Response, next: NextFunction) {
```

Comment:
No need this, this middleware is just a single line, just put in the respective functions